homeassistant/components/zwave_js/manifest.json

```diff
@@ -9,7 +9,7 @@
   "iot_class": "local_push",
   "loggers": ["zwave_js_server"],
   "quality_scale": "platinum",
-  "requirements": ["pyserial==3.5", "zwave-js-server-python==0.58.1"],
+  "requirements": ["pyserial==3.5", "zwave-js-server-python==0.59.0"],
```

Comment:
Please add a link to the release in the PR description.

We also should write a breaking change paragraph about what server version is required. See old PRs that bumped client version which required a new server schema version.

Reply:
The update is finally released everywhere. Let's go